### PR TITLE
Fix taking a crd that is not yet installed

### PR DESCRIPTION
**Pull request by Dmitrykob**
## Description

Added repeated queries to get crd and apply conversation strategies.

screen

## Why do we need it, and what problem does it solve?

The processing of conversation strategies occurs at the start of deckhouse and when a particular module is enabled.
In this case, processing may be started before crd is installed. This situation leads to an error.

## Why do we need it in the patch release (if we do)?

The problem described above occurs in the 1.58 update and users may encounter it.

## What is the expected result?

A good example to check is the multitenancy module.
It is necessary to turn off the module. Then remove the crd of this module such as (projects.deckhouse.io, projecttemplates.deckhouse.io) and enable the module again. The first attempt to apply conversation strategies will fail, but after a few attempts it will succeed.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: shell-operator
type: fix
summary: Repeat queries to get CRD and apply conversation strategies.
impact: Prevents a critical error from occurring in the cluster when starting or turning on modules.
impact_level: default
```

### CODE REVIEW

File: modules/002-deckhouse/docs/FAQ.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "The deckhouse module: FAQ"
+title: "The deckhouse module: FAQqw"
```
Comment (yalosev) [inline modules/002-deckhouse/docs/FAQ.md:2]:
```suggestion
title: "The deckhouse module: FAQ"
```

File: modules/000-common/images/shell-operator/werf.inc.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ shell:
     - make install-libLTLIBRARIES install-includeHEADERS
     - cp -f modules/oniguruma/src/.libs/libonig.* /libjq/lib
     - cd /
-    - git clone --branch {{ $shellOperatorVersion }} --depth 1 {{ .SOURCE_REPO }}/flant/shell-operator.git
+    - git clone --branch {{ $shellOperatorVersion }} --depth 1 https://github.com/flant/shell-operator.git
```
Comment (yalosev) [inline modules/000-common/images/shell-operator/werf.inc.yaml:53]:
```suggestion
    - git clone --branch {{ $shellOperatorVersion }} --depth 1 {{ .SOURCE_REPO }}/flant/shell-operator.git
```

File: modules/000-common/images/shell-operator/werf.inc.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{{- $shellOperatorVersion := "v1.4.5"}}
+{{- $shellOperatorVersion := "fix-on-startup-webhook-handler"}}
```
Comment (yalosev) [inline modules/000-common/images/shell-operator/werf.inc.yaml:1]:
```suggestion
{{- $shellOperatorVersion := "v1.4.6"}}
```

**Review by yalosev — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7835](https://github.com/deckhouse/deckhouse/pull/7835) to the branch [release-1.58](https://github.com/deckhouse/deckhouse/tree/release-1.58) successful!

**Pull request merged**